module.go

```diff
@@ -0,0 +1,40 @@
+package sandinghistorywebapp
+
+import (
+	"context"
+	"embed"
+	"io/fs"
+
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+
+	"github.com/erh/vmodutils"
+)
+
+//go:embed dist
+var staticFS embed.FS
+
+func DistFS() (fs.FS, error) {
+	return fs.Sub(staticFS, "dist")
+}
+
+var Model = resource.ModelNamespace("ncs").WithFamily("sanding-monitoring-web-app").WithModel("sanding-monitoring-web-app")
+
+func init() {
+	resource.RegisterComponent(
+		generic.API,
+		Model,
+		resource.Registration[resource.Resource, resource.NoNativeConfig]{
+			Constructor: newServer,
+		})
+}
+
+func newServer(ctx context.Context, deps resource.Dependencies, config resource.Config, logger logging.Logger) (resource.Resource, error) {
+	fs, err := DistFS()
```

Comment:
Let's simplify this down to the simplest version possible (ideally a no op generic server) given this isn't intended to ever actually be run & is only here to placate the viam build system.

Can you please:
1. File a bug ticket with the team that owns the github action stating that it is putting an unnecessary step on viam app authors to also build modules to get working ci
2. Add a comment in this file that links to that bug (along with all other files that only exist to placate the build system) that they can be deleted when that bug is fixed?